Google Play Music/UpdateDialog.cs

```diff
@@ -80,20 +80,19 @@ public UpdateDialog(string changeLog, string oldVersion, string newVersion)
 
         private void InitializeComponent()
         {
-            // Auto generated by the designer
-            notNowButton = new MaterialRaisedButton();
-            this.updateButton = new MaterialRaisedButton();
-            this.changeLogText = new MaterialLabel();
+            this.notNowButton = new MaterialSkin.Controls.MaterialRaisedButton();
+            this.updateButton = new MaterialSkin.Controls.MaterialRaisedButton();
+            this.changeLogText = new MaterialSkin.Controls.MaterialLabel();
```

Comment:
In the future using a program like [Source Tree](https://www.sourcetreeapp.com/) might help analyzing what you have changed and choosing what to commit to Version Control.  I would pull your fork and fix it but then you won't get the glorious commit credit :+1: 

I reckon you should (if you are unfamiliar with git you may have to google these terms)
* Squash the two commits you have into one 
* Manually revert the changes made to this file
* Commit that manual reversion with the `--amend` flag to apply it to your latest commit.
* Force push it up to GitHub (`git push -f`)

Hopefully that makes sense :)